app/Http/routes.php

```diff
@@ -79,6 +79,9 @@
 Route::get('/community/slack', ['as' => 'slack', 'uses' => 'CommunityController@getSlack']);
 Route::post('/community/slack/agree', ['as' => 'slack.agree', 'uses' => 'CommunityController@postSlackAgree']);
 
+Route::get('/mp/{match_id}', ['as' => 'multiplayer.match', 'uses' => 'MultiplayerController@match']);
+Route::get('/mp/{match_id}/history', ['as' => 'multiplayer.match.history', 'uses' => 'MultiplayerController@matchHistory']);
```

Comment:
i had something like that with beatmaps earlier (like `/beatmaps/{id}` and `/beatmapsets/{id}` redirecting into `/b/` and `/s/`), but eventually we just decided to go with the shorter ones, so i dunno